html/docs/taskwarrior.html

```diff
@@ -53,39 +53,56 @@
+              an <code>on-modify</code> hook script for Taskwarrior supplied by
+              Timewarrior. Simply copy it to the Taskwarrior hooks directory
+              and make it executable:
             </p>
 
+            <pre>cp /usr/local/share/doc/timew/ext/on-modify.timewarrior ~/.task/hooks/
+chmod +x ~/.task/hooks/on-modify.timewarrior</pre>
+
             <p>
-              The hook script is installed with Timewarrior.  Simply copy it to
-              the Taskwarrior hooks directory, and make it executable:
+              Depending on your system and installation method, you might need
+              to adjust the path you copy from.
             </p>
 
-            <pre>$ cp ext/on-modify.timewarrior ~/.task/hooks/
-$ chmod +x ~/.task/hooks/on-modify.timewarrior</pre>
-
             <p>
-              Once the hook is installed, run the Taskwarrior command:
+              Then run the Taskwarrior command <code>task diagnostics</code> to
+              check whether it was installed correctly. It should mention
+              <code>on-modify.timewarrior</code> in the Hooks > Active section:
             </p>
 
-            <pre>$ task diagnostics
-...</pre>
+            <pre><b>Hooks</b>
+     System: Enabled
+   Location: /home/user/.task/hooks
+     Active: on-modify.timewarrior (executable)</pre>
 
             <p>
-              This command will show whether the hook script is installed and
-              recognized. You should see something like this:
+              Whenever you <code>start</code> a task in Taskwarrior, the hook
+              script will start time tracking in Timewarrior. The hook captures
+              the description, project and tags of the task and passes them to
+              Timewarrior as tags.
+
+              When the task is <code>stop</code>ped, or completed, the hook
+              script will stop time tracking  in Timewarrior.
+
+              It works for any command that modifies the `start` property of
+              a task, but note that it currently does not detect if a newly
+              added task is immediately started. It also does not yet capture
             </p>
 
             <p>
-              <img src="/docs/images/hook1.png" class="img-responsive" />
+              The hook script is open source, written in Python, so feel free
+              to inspect and adjust it to your needs.
             </p>
 
             <p>
-              Whenever you <code>start</code> a task in Taskwarrior, the hook
-              script will start time tracking in Timewarrior. 
+              Below you can see an example usage: create a task, start it,
+              complete it and automatically track it.
+            </p>
 
-              When the task is <code>stop</code>ped, or completed, the hook
-              script will stop time tracking  in Timewarrior.
+            <p>
+                <img src="/docs/images/taskwarrior1.png" class="img-responsive" />
```

Comment:
I think a better name for this picture would be `taskwarrior-on-modify-hook-example.png`.